exporter/elasticsearchexporter/config.go

```diff
@@ -131,8 +131,12 @@ type DiscoverySettings struct {
 // all events already serialized into the send-buffer.
 type FlushSettings struct {
 	// Bytes sets the send buffer flushing limit.
+	// WARNING: This configuration is ignored. Use `flush.min_documents` instead.
```

Comment:
Can we also add a deprecated notice here? Something like: `deprecated: [v0.xxx.xx] Use `flush.min_documents` instead.`

Reply:
Not sure if I know in which version this change will be released. How useful is the version in the deprecation comment?

Reply:
> Not sure if I know in which version this change will be released.

Won't this be the next version (one after the current one)? Or are you thinking if a major version change happens in the next one?

> How useful is the version in the deprecation comment?

If you are looking at a future version of the code then knowing which version a particular config got deprecated is useful.

_(This point is more like a nit so feel free to make a decision as you see fit)_